Common/VR/VRBase.cpp

```diff
@@ -64,6 +153,13 @@ void VR_Init( ovrJava java ) {
 		exit(1);
 	}
 
+#ifdef OPENXR_PLATFORM_PICO
+	InitializeGraphicDeivce(vr_engine.appState.Instance);
+	xrGetInstanceProcAddr(vr_engine.appState.Instance,"xrSetConfigPICO", (PFN_xrVoidFunction*)(&pfnXrSetConfigPICO));
+	Pxr_SetEngineVersion("2.8.0.1");
```

Comment:
Is this intentionally hardcoded or is it meant to be related to the PPSSPP version?

-[Unknown]

Reply:
This is intentionally hardcoded, as a developer you can choose which OpenXR implementation version you want to use. I got this from the SDK sample code.